### ISSUE TITLE
Bug: notifications appear to be delayed by 30 minutes

### ISSUE THREAD

**Issue by jwjenkin** [hbrennhaeuser/homeassistant_integration_ntfy#3]
Haven't had time to dig into why, but there appears to be a severe lag (about 30 minutes) between when a notification is triggered in HA and when it is sent.

Can confirm it's related to this, as I have other services that send near instant notifications, and curling the ntfy server also works near real-time.

**Comment by hbrennhaeuser** [comment]
Thanks for reporting the issue. 
I just checked my HA and ntfy-notifications don't seem to be working at all any more for me.
I guess the issue is caused by one of the latest HA-updates, since notifications were working a few weeks ago.

Can you tell me how you are running your Home Assistant and what version(s) you are on?
I'm currently running HA OS Generic x86-64, Core 2023.5.3, OS  10.0.

I'll have a look at the problem and report back.

**Comment by hbrennhaeuser** [comment]
My problem was not related to the integration, rather to an error in a firewall rule. 
Also I updated my setup to Core 2023.5.4, OS 10.2. Notifications work fine for me (now).

To test the integration, I created an empty automation that just triggers a notification. When I run the automation (manually), I instantly and reliably get the notification on my phone.

Have you tried triggering a notification through a basic automation to rule out any possible logic errors in your automations? Are there any ntfy related error messages in the HA-logs?
